libsql-sqlite3/src/wal.c

```diff
@@ -581,7 +596,7 @@ struct WalIterator {
 ** Scenario (3) can only occur when pWal->writeLock is false and iPage==0
 */
 static SQLITE_NOINLINE int walIndexPageRealloc(
-  Wal *pWal,               /* The WAL context */
+  struct sqlite3_wal *pWal,               /* The WAL context */
```

Comment:
That was just a typedef, why change it? Let's keep it as is to minimize the changes in this already large patch